src/plan/global.rs

```diff
@@ -328,6 +328,10 @@ pub trait Plan: 'static + Sync + Send + Downcast {
             );
         }
     }
+
+    fn global_side_metadata_per_chuck(&self) -> usize {
```

Comment:
```suggestion
    fn global_side_metadata_per_chunk(&self) -> usize {
```

Reply:
I don't understand what has changed here

Reply:
I assume it is a typo. Is it 'per_chunk' or 'per_chuck'?

Reply:
right, it's a typo, per_chunk is correct